src/views/Intervention/Intervention.jsx

```diff
@@ -252,26 +256,26 @@ const Intervention = ({ routeParamMap, routePath, data }) => {
 		const pagerOffset = getPageOffset(page, itemsPerPage);
 		return (
 			<>
-				{trialsPayload?.trials?.length > 0 && (
+				{fetchedAndHasTrials(fetchState) && (
```

Comment:
This code is crazy, @arcepaul and I discussed it and we really should not have so many things "rendering" above and below the spinner if they do not show during load.